stubs/setuptools/METADATA.toml

```diff
@@ -1,9 +1,10 @@
 version = "75.6.*"
 upstream_repository = "https://github.com/pypa/setuptools"
 extra_description = """\
-If using `setuptools >= 71.1` *only* for `pkg_resources`,
-you don't need `types-setuptools` since `pkg_resources` is now typed.\
+Given that `pkg_resources` is typed since `setuptools >= 71.1`, \
+it is no longer included with `types-setuptools`.
 """
+requires = ["setuptools"] # For pkg_resources
```

Comment:
Oh, a stub can't depend on the library it stubs >.<
Since this restriction is in place for security reasons, would it make sense to allow `types-Foo` to require `Foo` ?
> stub_uploader.metadata.InvalidRequires: Expected dependency setuptools to be listed in setuptools's requires_dist or the sdist's *.egg-info/requires.txt